ivy/functional/backends/paddle/elementwise.py

```diff
@@ -482,4 +482,4 @@ def trunc_divide(
 def isreal(
         x: paddle.Tensor, /, *, out: Optional[paddle.Tensor] = None
 ) -> paddle.Tensor:
-    return paddle.equal(x, paddle.real(x))
+    return paddle.equal(x, paddle.real(x))
```

Comment:
paddle.real only supports complex64 and complex128, so please handle this case, maybe if the input tensor isn't of these types, return true for all values